contrib/packs/actions/download.yaml

```diff
@@ -5,23 +5,16 @@
   enabled: true
   entry_point: "pack_mgmt/download.py"
   parameters:
-    packs:
-      type: "array"
-      items:
-        type: "string"
-    repo_url:
+    pack:
```

Comment:
You absolutely need a description to `pack`. I looked down to code and looks like it may be quite a few things :)